packages/core/android/gradle.properties

```diff
@@ -1 +1,11 @@
-emb_android_sdk=6.13.0
\ No newline at end of file
+android.useAndroidX=true
+
+emb_android_sdk=6.13.0
```

Comment:
this is no longer needed right since we're reading it from the package json now?